src/wp-includes/block-bindings/sources/post-meta.php

```diff
@@ -0,0 +1,37 @@
+<?php
+/**
+ * Add the post_meta source to the Block Bindings API.
+ *
+ * @since 6.5.0
+ * @package WordPress
+ */
+function post_meta_source_callback( $source_attrs ) {
```

Comment:
Why this source callback has only one argument but the pattern override one has 3? Shouldn't they work similarly?

Reply:
I think it should use at least two params, the second one is `$block_instance` where we should read context as noted in the code and GitHub comments. The third parameter is the name of the attribute that gets updated. I was on the fence with that one, as only Pattern Overrides need it today when processing the block context.

Reply:
Yes, after trying to understand more, I'm not too concerned at the moment.